sklearn/utils/tests/test_extmath.py

```diff
@@ -457,6 +458,17 @@ def test_incremental_variance_ddof():
                                 calculated_variances, 6)
             assert_equal(incremental_count, sample_count)
 
+
+def test_eigen_sign_flip():
+    # Testing that sign flip is working & largest value has positive sign
+    data = np.random.RandomState(36).randn(5, 5)
+    max_abs_rows = np.argmax(np.abs(data), axis=1)
+    data_flipped = _deterministic_vector_sign_flip(data)
+    max_rows = np.argmax(data_flipped, axis=1)
+    assert_array_equal(max_abs_rows, max_rows)
+    signs = np.sign(data[range(data.shape[0]), max_abs_rows])
+    assert_array_equal(data, data_flipped*signs[:, np.newaxis])
+
```

Comment:
I will do the changes myself when merging but FYI PEP8 requires 2 blank lines to separate top level blocks and white spaces around binary operators.